.cdsrc.json

```diff
@@ -1,11 +1 @@
-{
-    "sql": {
-        "transitive_localized_views" : false,
-        "native_hana_associations" : false
-    },
-    "cdsc": {
-        "fewerLocalizedViews": true,
-        "betterSqliteSessionVariables": true,
```

Comment:
@johannes-vogel Isn't that one needed for the node runtime with the new sqlite DB service?

Reply:
```sh
cds deploy --dry -2 sqlite

->

... 

CREATE VIEW localized_sap_fe_cap_travel_SupplementType AS SELECT
  coalesce(localized_1.name, L_0.name) AS name,
  coalesce(localized_1.descr, L_0.descr) AS descr,
  L_0.code
FROM (sap_fe_cap_travel_SupplementType AS L_0 LEFT JOIN sap_fe_cap_travel_SupplementType_texts AS localized_1 ON localized_1.code = L_0.code AND localized_1.locale = session_context( '$user.locale' )); 

...
```

Looks good to me.

Also `cds run` starts a CAP Node.js app, which looks OK to me.
```